bind9/README.md

```diff
@@ -1,77 +1,229 @@
 
-## Setup
 
-The Bind9 check is not included in the [Datadog Agent][2] package, so you need to install it.
+## Setup
 
 ### Installation
 
-For Agent v7.21+ / v6.21+, follow the instructions below to install the Bind9 check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
-
-1. Run the following command to install the Agent integration:
-
-   ```shell
-   datadog-agent integration install -t datadog-bind9==<INTEGRATION_VERSION>
-   ```
-
-2. Configure your integration similar to core [integrations][4].
+To install the Bind9 integration, run the following Agent installation command and the steps below. For more information, see the [Integration Management][14] documentation.
+
+**Note**: This step is not necessary for Agent version >= 7.58.0.
+
+Linux command
+  ```shell
+  sudo -u dd-agent -- datadog-agent integration install datadog-bind9==1.1.0
+  ```
+
+#### Log collection
+
+#### File Monitoring
+
+1. Log in to your Bind9 device.
+2. Open `named.conf` file to add a logging clause:
+    ```
+    logging {
+     channel <example_channel> {
+          file "/folder_path/file_name.log" versions <unlimited | <integer>> size <size> suffix <increment | timestamp>;
+          print-time (yes | local | iso8601 | iso8601-utc);
+          print-category yes;
+          print-severity yes;
+     };
+     category <example-category> { <example_channel>; };
+    }
+    ```
+    **NOTE**: Recommended value for `print-time` is `iso8601-utc` because datadog expects all logs to be in the UTC time zone by default. If the timezone of your Bind9 logs is not UTC please make sure to follow [these](#timezone-steps) steps. Also, check the categories defined by Bind9 [here][16].
+    
+    Example logging channel:
+    ```
+    logging {
+     channel default_log {
+          file "/var/log/named/query.log" versions 3 size 10m;
+          print-time iso8601-utc;
+          print-category yes;
+          print-severity yes;
+     };
+       category default { default_log; };
+    }
+    ```
+3. Save and exit the file.
+4. Restart the service
+    ```
+    service named restart
+    ```
+
+#### Syslog
+1. Log in to your Bind9 device.
+2. Open `named.conf` file to add a logging clause:
+    ```
+    logging {
+     channel <example_channel> {
+          syslog <syslog_facility>;
+          severity (critical | error | warning | notice | info | debug [level ] | dynamic);
+          print-time (yes | local | iso8601 | iso8601-utc);
+          print-category yes;
+          print-severity yes;
+     };
+     category <example-category> { <example_channel>; };
+    }
+    ```
+    **NOTE**: Recommended value for `print-time` is `iso8601-utc` because Datadog expects all logs to be in the UTC time zone by default. If the timezone of your Bind9 logs is not UTC please make sure to follow [these](#timezone-steps) steps. Also, check the categories defined by Bind9 [here][16].
```

Comment:
```suggestion
    **NOTE**: Recommended value for `print-time` is `iso8601-utc` because Datadog expects all logs to be in the UTC time zone by default. If the timezone of your Bind9 logs is not UTC please make sure to follow [the steps for using a different time zone](#timezone-steps). Also, [check the categories defined by Bind9][16].
```

Reply:
Done